src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -191,7 +192,7 @@ public Task<string> ReadAsStringAsync()
                 Encoding encoding = null;
                 int bomLength = -1;
 
-                byte[] data = innerThis.GetDataBuffer(innerThis._bufferedContent);
+                byte[] data = innerThis._bufferedContent.ToArray();
```

Comment:
Why did this change to ToArray?  Can TryGetBuffer not be used here?  If this does change to ToArray, the below comment is now invalid.